shared/translations/languages/fr.json

```diff
@@ -693,7 +703,8 @@
     "Swapping Token": "Échange du token",
     "Swapping {{amountA}} {{symbolA}} to {{amountB}} {{symbolB}}": "Échange de {{amountA}} {{symbolA}} avec {{amountB}} {{symbolB}}",
     "Slippage tolerance": "Tolérance de glissement",
-    "Estimated to receive": "Estimé à recevoir"
+    "Estimated to receive": "Estimé à recevoir",
+    "Swap on future block {{amountA}} {{fromTokenDisplaySymbol}} to {{toTokenDisplaySymbol}} on oracle price {{percentageChange}}": "Swap on future block {{amountA}} {{fromTokenDisplaySymbol}} to {{toTokenDisplaySymbol}} on oracle price {{percentageChange}}"
```

Comment:
"Swap on future block {{amountA}} {{fromTokenDisplaySymbol}} to {{toTokenDisplaySymbol}} on oracle price {{percentageChange}}": "Échanger sur le bloc futur des {{amountA}} {{fromTokenDisplaySymbol}} contre {{toTokenDisplaySymbol}} sur le prix oracle {{percentageChange}}"